### DISCUSSION TITLE
AutoGenerateAttribute vs AutoGeneratedAttribute

### DISCUSSION THREAD

**Discussion by meeech** [typedorm/typedorm#164] [Q&A] [answered]
👋🏼 @whimzyLive I notice in the docs you make reference to `AutoGeneratedAttribute`, but it seems the decorator is `AutoGenerateAttribute`. 
I'm assuming you would want the docs updated to show using `AutoGenerateAttribute` as opposed to changing the name of the decorator to `AutoGeneratedAttribute`. 
Just wanted to check in with you before I do that - i'm building up a PR of small doc fixes as I work through getting typedorm setup in my project.

let me know

**Comment by whimzyLive** [comment, marked as answer]
@meeech  Thanks for reporting this. Yes, it should definitely be `@AutoGenerateAttribute`. 

P.S. I am going to convert this discussion to an issue so that it can be better tracked. 
Can you please ping [here](https://github.com/typedorm/typedorm/issues/165), when you have started working on the correction?

**Comment by meeech** [comment]
Sounds good.  Feel free to assign it to me. I will take care of it in my pr. 

M

**Reply by whimzyLive** [reply]
Yep, But GitHub won't let me assign it to you until you have interacted with the issue. :)

**Reply by whimzyLive** [reply]
Closing this discussion as this has already been resolved.
